batavia/builtins/hash.js

```diff
@@ -1,8 +1,8 @@
 var exceptions = require('../core').exceptions
 var type_name = require('../core').type_name
-var types = require('../types')
 
 function hash(args, kwargs) {
+    var types = require('../types')
```

Comment:
Is this change intended?

Node.js advises against `require` inside function body:
https://github.com/alanjames1987/Node.js-Best-Practices/issues/3